### PR TITLE
refactor project structure

### PR DESCRIPTION
**Pull request by zaro0508**
Refactor to more closely align with the ofn reference project
structure[1].

* create separate folders for ofn resource providers
* move scp out of organizations.yaml and into it's own folder

[1] https://github.com/org-formation/org-formation-reference

### CODE REVIEW

File: org-formation/organization.yaml [hunk 1]
```diff
@@ -444,60 +444,3 @@ Organization:
                 'aws:RequestedRegion':
                   - us-east-1
                   - us-east-2
```
Comment (ahayden) [inline org-formation/organization.yaml:446]: Do you mean to include `us-east-2`? I see that its in the sid name on L436 too

**Review by ahayden — APPROVED** [review]
Nice

File: org-formation/005-types/_tasks.yaml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+PasswordPolicyRp:
+  Type: register-type
+  ResourceType: 'Community::IAM::PasswordPolicy'
+  SchemaHandlerPackage: !Sub 's3://${catalogBucket}/community-iam-passwordpolicy-0.2.0.zip'
+  MaxConcurrentTasks: 10
+  OrganizationBinding:
+    IncludeMasterAccount: true
+    Account: '*'
+    Region: !Ref primaryRegion
```
Comment (ahayden) [inline org-formation/005-types/_tasks.yaml:75]: Why is this only on primary region?

File: org-formation/005-types/_tasks.yaml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    Region: !Ref allRegions
+
+SsoAssignmentGroupRp:
+  Type: register-type
+  ResourceType: 'Community::SSO::AssignmentGroup'
+  SchemaHandlerPackage: !Sub 's3://${catalogBucket}/community-sso-assignmentgroup-0.3.1.zip'
+  MaxConcurrentTasks: 10
+  OrganizationBinding:
+    IncludeMasterAccount: true
+    Region: !Ref primaryRegion
```
Comment (ahayden) [inline org-formation/005-types/_tasks.yaml:114]: Same as above... why only primary?

File: org-formation/organization.yaml (hunk 1 above)
Comment (zaro0508) [reply org-formation/organization.yaml:446]: actually those were read by ofn and included in this file.  I'm going to move all of those SCPs into the `010-scps` folder like i did with RestrictUnusedRegionsSCP.  will do it in follow on PR.

File: org-formation/005-types/_tasks.yaml (hunk 3 above)
Comment (zaro0508) [reply org-formation/005-types/_tasks.yaml:114]: i just copied these settings from the ofn reference project.  I currently have primaryRegion and all AllRegions both set to us-east-1 right now.  so it's the same anyways

**Pull request merged**